### PR TITLE
Jennyf/fixes

### PR DESCRIPTION
**Pull request by jennyf19**
Ran into some issues testing w/msal desktop app...addressing those issues here.

### CODE REVIEW

File: msal/src/Microsoft.Identity.Client/Account.cs [hunk 1]
```diff
@@ -61,11 +60,20 @@ public Account(AccountId homeAccountId, string username, string environment)
 
         public override string ToString()
         {
-            return String.Format(
+            if (HomeAccountId != null)
+            {
+                return String.Format(
                 CultureInfo.CurrentCulture,
                 "Account username: {0} environment {1} home account id: {2}",
                 Username, Environment, HomeAccountId.ToString());
```
Comment (jmprieur) [inline msal/src/Microsoft.Identity.Client/Account.cs]: you don't need the ToString() on HomeAccountId

**Review by jmprieur — APPROVED** [review]
LGTM

File: msal/src/Microsoft.Identity.Client/Account.cs [hunk 2]
```diff
@@ -61,11 +60,20 @@ public Account(AccountId homeAccountId, string username, string environment)
 
         public override string ToString()
         {
-            return String.Format(
```
Comment (bgavrilMS) [inline msal/src/Microsoft.Identity.Client/Account.cs]: It looks like the parameterless ctor is only used by tests. Could we get rid of it instead? This way HomeAccountId would be guaranteed not null.

File: msal/src/Microsoft.Identity.Client/Account.cs (hunk 1 above)
Comment (jennyf19) [reply msal/src/Microsoft.Identity.Client/Account.cs]: removed. thanks.

File: msal/src/Microsoft.Identity.Client/Account.cs (hunk 2 above)
Comment (jennyf19) [reply msal/src/Microsoft.Identity.Client/Account.cs]: great idea. thanks.

Comment (jennyf19) [reply msal/src/Microsoft.Identity.Client/Account.cs]: @bgavrilMS I updated all the unit tests.

**Review by bgavrilMS — APPROVED** [review]
very cool, LGTM

**Pull request merged**